src/main/java/org/jabref/logic/importer/fetcher/INSPIREFetcher.java

```diff
@@ -59,33 +56,38 @@ public String getName() {
     @Override
     public URL getURLForQuery(String query) throws URISyntaxException, MalformedURLException, FetcherException {
         URIBuilder uriBuilder = new URIBuilder(INSPIRE_HOST);
-        uriBuilder.addParameter("p", query); // Query
-        //uriBuilder.addParameter("jrec", "1"); // Start index (not needed at the moment)
-        uriBuilder.addParameter("rg", "100"); // Should return up to 100 items (instead of default 25)
-        uriBuilder.addParameter("of", "hx"); // BibTeX format
+        uriBuilder.addParameter("q", query); // Query
         return uriBuilder.build().toURL();
     }
 
     @Override
-    public Parser getParser() {
-        // Inspire returns the BibTeX result embedded in HTML
-        // So we extract the BibTeX string from the <pre>bibtex</pre> tags and pass the content to the BibTeX parser
-        return inputStream -> {
-            String response = new BufferedReader(new InputStreamReader(inputStream)).lines().collect(Collectors.joining(OS.NEWLINE));
+    public List<BibEntry> performSearch(String query) throws FetcherException {
+        if (StringUtil.isBlank(query)) {
```

Comment:
Instead of copy-pasting the method, I would extract `new URLDownload(getURLForQuery(query)).asInputStream()` from https://github.com/JabRef/jabref/blob/master/src/main/java/org/jabref/logic/importer/SearchBasedParserFetcher.java#L58 to a new default method, which you then can overwrite here.